static/task-management-template/index.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Task Management Template</title>
+  </head>
+  <body>
+
+    <!-- navbar -->
+
+    <nav class="">
+      <div class="max-w-6xl mx-auto px-4">
+        <div class="flex justify-between">
+          <div class="sm:flex sm:items-center space-x-7 w-full">
+
+            <!-- Website Logo -->
+
+            <div>
+              <div class="flex items-center justify-between py-4 px-2">
+                <a href="#">
+                  <img src=".\assets\Logo.svg" alt="Logo" class="h-8 w-30 mr-2" />
+                </a>
+                <button class="sm:hidden">
+                  <img class="h-6 w-6" src=".\assets\hamburger.svg" alt="">
+                </button>
+              </div>
+            </div>
+
+            <!-- Primary Navbar items -->
+
+            <div class="hidden md:flex items-center space-x-2 px-40">
+              <a href="" class="py-4 px-2 text-blue-500 border-b-4 border-blue-500 font-semibold ">
+                Features
+              </a>
+
+              <a href="" class="py-4 px-2 text-gray-500 font-semibold hover:text-blue-500 transition duration-300">
+                Pricing
+              </a>
+
+              <a href="" class="py-4 px-2 text-gray-500 font-semibold hover:text-blue-500 transition duration-300">
+                Resources
+              </a>
+              
+            </div>
+            
+            <!-- login buttons -->
+
+            <div class="hidden sm:block">
+              <button class="bg-color-white hover:bg-grey-500 font-semibold px-4 py-2 rounded">Log In</button>
+              <button class="bg-blue-500 hover:bg-blue-700 text-white font-semibold px-4 py-2 rounded">Sign up</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+
+    <!-- landing page -->
```

Comment:
Main content should be wrapped in <main> element

Reply:
I have done but not sure if I did it right